app/utils/tsinterval.js

```diff
@@ -0,0 +1,13 @@
+export default function tsinterval() {
+    let d1 = new Date(this.get('gte'));
```

Comment:
yes i should probably get dates as parameters, otherwise not very flexible as a utility function.